boards/arm/wio_terminal/pinmux.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2020 Google LLC.
+ * Copyright (c) 2021 Szymon Duchniewicz
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <init.h>
+#include <drivers/pinmux.h>
+#include <soc.h>
+
+static int board_pinmux_init(const struct device *dev)
+{
+	__unused const struct device *muxa = DEVICE_DT_GET(DT_NODELABEL(pinmux_a));
+	__unused const struct device *muxb = DEVICE_DT_GET(DT_NODELABEL(pinmux_b));
+
+	__ASSERT_NO_MSG(device_is_ready(muxa));
+	__ASSERT_NO_MSG(device_is_ready(muxb));
+
+	ARG_UNUSED(dev);
+	ARG_UNUSED(muxa);
```

Comment:
I don't think muxa/muxb need to be marked ARG_UNUSED, since they aren't arguments.